### PR TITLE
HDDS-5569. recipe/BotoClient.md translation

### PR DESCRIPTION
**Pull request by will-sh**
## What changes were proposed in this pull request?
HDDS-5569. recipe/BotoClient.md translation to Chinese

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-5569

## How was this patch tested?
hugo serve

### CODE REVIEW

File: hadoop-hdds/docs/content/recipe/BotoClient.zh.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
+---
+title: 使用 Boto3 客户端访问 Ozone 对象存储
+linktitle: Boto3
+summary: 如何使用 Boto3 客户端访问Ozone对象存储 ？
```
Comment (smengcl) [inline hadoop-hdds/docs/content/recipe/BotoClient.zh.md:4]:
```suggestion
summary: 如何使用 Boto3 客户端访问 Ozone 对象存储？
```

File: hadoop-hdds/docs/content/recipe/BotoClient.zh.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+- Upload file
+- Download file
+- Delete objects(keys)
+- Head object
+- Multipart upload
+
+
+## 要求
+
+您将需要更高版本的 Python3 作为您的 Boto3 客户端，因为 Boto3 安装要求在此处指出:
```
Comment (smengcl) [inline hadoop-hdds/docs/content/recipe/BotoClient.zh.md:38]: Translating as-is sounds .. stiff. Touching up a bit

```suggestion
您将需要较高版本的 Python3 来运行 Boto3 客户端，请参考 Boto3 的安装需求:
```

File: hadoop-hdds/docs/content/recipe/BotoClient.zh.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+        s3_client = session.client(
+            service_name='s3',
+            aws_access_key_id='testuser/scm@EXAMPLE.COM',
+            aws_secret_access_key='c261b6ecabf7d37d5f9ded654b1c724adac9bd9f13e247a235e567e8296d2999',
+            endpoint_url='http://localhost:9878',
+        )
+        'endpoint_url' is pointing to Ozone s3 endpoint.
+
+    In our code sample below, we're demonstrating the usage of both s3 and s3_client.
```
Comment (smengcl) [inline hadoop-hdds/docs/content/recipe/BotoClient.zh.md:71]: missing new line

```suggestion
    In our code sample below, we're demonstrating the usage of both s3 and s3_client.

```

File: hadoop-hdds/docs/content/recipe/BotoClient.zh.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+这将在一个名为 `s3v` 的卷中创建一个名为 `bucket1` 的桶
+
+### 列出所有桶
+    response = s3_client.list_buckets()
+    print('Existing buckets:')
+    for bucket in response['Buckets']:
+        print(f'  {bucket["Name"]}')
+
+这将列出Ozone `s3v` 卷中的所有桶
```
Comment (smengcl) [inline hadoop-hdds/docs/content/recipe/BotoClient.zh.md:90]:
```suggestion
这将列出 Ozone `s3v` 卷中的所有桶
```

File: hadoop-hdds/docs/content/recipe/BotoClient.zh.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+    response = s3_client.delete_bucket(Bucket='bucket1')
+    print(response)
+
+这将从 Ozone 卷 `s3v` 中删除一个桶 `bucket1`。
+
+### 上传文件
+    response = s3.Bucket('bucket1').upload_file('./README.md','README.md')
+    print(response)
+
+这将从向Ozone卷 `s3v` 和桶 `bucket1` 中上传一个 `README.md` 文件并创建一个 `README.md` 键。
```
Comment (smengcl) [inline hadoop-hdds/docs/content/recipe/BotoClient.zh.md:108]:
```suggestion
这将从向 Ozone 卷 `s3v` 和桶 `bucket1` 中上传 `README.md` 文件并创建一个 `README.md` 键。
```

File: hadoop-hdds/docs/content/recipe/BotoClient.zh.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+
+### 下载文件
+    response = s3.Bucket('bucket1').download_file('README.md', 'download.md')
+    print(response)
+
+这将从从Ozone卷 `s3v` 和桶 `bucket1` 中下载 `README.md` 并创建一个 `README.md` 文件到本地。
```
Comment (smengcl) [inline hadoop-hdds/docs/content/recipe/BotoClient.zh.md:114]:
```suggestion
这将从从 Ozone 卷 `s3v` 和桶 `bucket1` 中下载 `README.md` 并创建一个 `README.md` 文件到本地。
```

File: hadoop-hdds/docs/content/recipe/BotoClient.zh.md [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+This will download 'README.md' from Ozone volume 's3v' to local and create a file with name 'download.md'.
```
Comment (smengcl) [inline hadoop-hdds/docs/content/recipe/BotoClient.zh.md:115]:
```suggestion
```

File: hadoop-hdds/docs/content/recipe/BotoClient.zh.md [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+
+### 查看对象信息
+    response = s3_client.head_object(Bucket='bucket1', Key='README.md')
+    print(response)
+
+这将查看一个位于Ozone 卷 `s3v` 和桶 `bucket1` 中的 `README.md` 文件的信息。
```
Comment (smengcl) [inline hadoop-hdds/docs/content/recipe/BotoClient.zh.md:121]:
```suggestion
这将查看一个位于 Ozone 卷 `s3v` 和桶 `bucket1` 中的 `README.md` 文件的信息。
```

File: hadoop-hdds/docs/content/recipe/BotoClient.zh.md [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+                },
+                {
+                    'Key': 'README3.md',
+                },
+            ],
+            'Quiet': False,
+        },
+    )
+
+这将从Ozone卷 `s3v` 和桶 `bucket1` 中删除多个对象 `README3.md` 和 `README4.md`
```
Comment (smengcl) [inline hadoop-hdds/docs/content/recipe/BotoClient.zh.md:139]:
```suggestion
这将从 Ozone 卷 `s3v` 和桶 `bucket1` 中删除多个对象 `README3.md` 和 `README4.md`
```

**Review by smengcl — APPROVED** [review]
Thanks @will-sh for the translation.

**Comment by will-sh** [comment]
thx for review @smengcl

**Comment by smengcl** [comment]
Thanks @will-sh for the PR.

**Comment by smengcl** [comment]
I think there is a Github glitch that prevents it from marking this PR as Merged. It is already merged in `master`:

https://github.com/apache/ozone/commit/5d999d2068618b9e1e8ed66188b3ea22dc225d1d

I will leave this PR here for a few hours to see if it fixes itself. If not I will close this manually.

**Comment by adoroszlai** [comment]
Thanks @will-sh for the translation, @smengcl for the review.

**Pull request closed without merging**